Classes/Domain/Repository/FormDataRepository.php

```diff
@@ -108,17 +108,29 @@ public function findFormDataWithIdentifierContainingData(string $formIdentifier,
     public function createQueryBuilder($alias, $indexBy = null)
     {
         $queryBuilder = parent::createQueryBuilder($alias, $indexBy);
-        return $queryBuilder->andWhere($queryBuilder->expr()->in('form.siteName', $this->accessibleSites));
+        if (count($this->accessibleSites) > 0) {
+            return $queryBuilder->andWhere($queryBuilder->expr()->in('form.siteName', $this->accessibleSites));
+        }
+
+        return $queryBuilder->andWhere('2 = 1');
+    }
+
+    /**
+     * @return string[]
```

Comment:
This shows the wrong return type.

Reply:
no, it doesn't ? It returns an array of strings.